### PR TITLE
H3 draft 24

### PR DESCRIPTION
**Pull request by stammw**
#520

### CODE REVIEW

File: quinn-h3/src/body.rs [hunk 1]
```diff
@@ -62,6 +62,7 @@ pub struct BodyReader {
     finish_request: bool,
 }
 
+// TODO don't forget to rebase reorder
```
Comment (djc) [inline quinn-h3/src/body.rs]: Don't forget!

**Comment by djc** [comment]
And, it should get rebased. Otherwise, looks good to me!

File: quinn-h3/src/body.rs (hunk 1 above)
Comment (stammw) [reply quinn-h3/src/body.rs]: I forgot to put a comment to remind this comment removal.

Comment (stammw) [reply quinn-h3/src/body.rs]: There it is !

**Pull request merged**